tests/vm/gh_test.go

```diff
@@ -128,9 +128,15 @@ func RunVmTest(p string, t *testing.T) {
 			ret, logs, gas, err = helper.RunState(statedb, env, test.Transaction)
 		}
 
-		rexp := helper.FromHex(test.Out)
-		if bytes.Compare(rexp, ret) != 0 {
-			t.Errorf("%s's return failed. Expected %x, got %x\n", name, rexp, ret)
+		switch name {
+		// the memory required for these tests (4294967297 bytes) would take too much time.
+		// on 19 May 2015 decided to skip these tests their output.
```

Comment:
clarify that it's the validation of the returned value which would take too much time and memory